ratatool-extras/src/main/scala/com/spotify/ratatool/io/ParquetIO.scala

```diff
@@ -40,7 +22,7 @@ object ParquetIO {
     val reader = AvroParquetReader.builder[T](path)
       .withConf(conf)
       .build()
-      .asInstanceOf[ParquetReader[T]]
+
```

Comment:
was this change intentional?

Reply:
yeah the `asInstanceOf` wasn't necessary.  Doesn't make a big difference either way though